src/screens/OtherProfile/index.js

```diff
@@ -1042,39 +897,15 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
     alignItems: 'center'
   },
-  toggleSwitchContainer: {display: 'flex', alignSelf: 'flex-end', paddingVertical: 10},
-  rightHeaderContentContainer: {display: 'flex', flexDirection: 'row'},
-  headerImageContainer: {display: 'flex', flexDirection: 'row', marginBottom: 20},
-  bioAndSendChatContainer: (dynamicColors) => ({
-    backgroundColor: dynamicColors.primary,
-    borderRadius: 15,
-    paddingHorizontal: 10,
-    paddingTop: 10
-  }),
-  bioText: (dynamicColors) => ({
-    color: dynamicColors.text,
-    fontSize: 14,
-    fontWeight: '600',
-    lineHeight: 22
-  })
+  rightHeaderContentContainer: {
+    display: 'flex',
+    flexDirection: 'row'
+  },
+  headerImageContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    marginBottom: 20
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New styles `rightHeaderContentContainer` and `headerImageContainer` have been added. Ensure that these styles are applied correctly in the component.
```diff
+ rightHeaderContentContainer: {
+   display: 'flex',
+   flexDirection: 'row'
+ },
+ headerImageContainer: {
+   display: 'flex',
+   flexDirection: 'row',
+   marginBottom: 20
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->